### PR TITLE
Refine api docs and readme and fix baseline bug

### PR DESCRIPTION
**Pull request by smallv0221**
### PR types

Others
### PR changes

Models
### Description

Refine api docs and readme and fix baseline bug

### CODE REVIEW

File: examples/text_generation/unimo-text/README.md [hunk 1]
```diff
@@ -91,54 +114,27 @@ python -m paddle.distributed.launch --gpus "0" --log_dir ./log run_gen.py \
 
 ### 模型预测
 
-运行如下命令即可在样例测试集上进行测试
+本基线还提供了一个finetune好的unimo-text-1.0模型，可以从xx下载。
```
Comment (guoshengCS) [inline examples/text_generation/unimo-text/README.md]: 这里后面注意更新

File: examples/text_generation/unimo-text/README.md [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
 
 - 文案生成根据结构化的商品信息生成合适的广告文案；
 - 摘要生成是为输入文档生成简洁且包含关键信息的简洁文本；
 - 问题生成则是根据给定段落以及答案生成适合的问题。
 
+为了方便用户快速使用基线，PaddleNLP Dataset API内置了数据集，一键即可完成数据集加载，示例代码如下：
+
+```python
+from paddlenlp.datasets import load_dataset
+train_ds, dev_ds = load_dataset('dureader_qg', splits=('train', 'dev'))
```
Comment (ZeyuChen) [inline examples/text_generation/unimo-text/README.md:30]: 得读音新增dataset文档介绍

File: examples/text_generation/unimo-text/README.md [hunk 3]
```diff
@@ -91,54 +114,27 @@ python -m paddle.distributed.launch --gpus "0" --log_dir ./log run_gen.py \
...
+python run_gen.py \
+    --dataset_name=dureader_qg \
+    --model_name_or_path=your_model_path \
+    --logging_steps=100 \
+    --batch_size=16 \
+    --max_seq_len=512 \
+    --max_target_len=30 \
+    --do_predict \
+    --max_dec_len=20 \
+    --max_dec_len=3 \
```
Comment (guoshengCS) [inline examples/text_generation/unimo-text/README.md]: 下面这里是否是`min_dec_len`呢

File: examples/text_generation/unimo-text/README.md (hunk 2 above)
Comment (smallv0221) [reply examples/text_generation/unimo-text/README.md:30]: done

File: examples/text_generation/unimo-text/README.md (hunk 1 above)
Comment (smallv0221) [reply examples/text_generation/unimo-text/README.md]: done

File: examples/text_generation/unimo-text/README.md (hunk 3 above)
Comment (smallv0221) [reply examples/text_generation/unimo-text/README.md]: done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**